packages/@react-aria/combobox/docs/useComboBox.mdx

```diff
@@ -166,21 +176,20 @@ function ComboBox(props) {
         {...inputProps}
         ref={inputRef}
         style={{
-          borderRight: 0,
-          borderBottomRightRadius: 0,
-          borderTopRightRadius: 0
+          height: 22,
+          boxSizing: 'border-box',
+          marginRight: 0
         }} />
       <button
         {...buttonProps}
         ref={triggerRef}
         style={{
-          height: '21px',
-          borderBottomLeftRadius: 0,
-          borderTopLeftRadius: 0
```

Comment:
Just wanna double check, but do we not want these radius changes? Makes the input and button look like two separate elements instead of a single combobox

Reply:
Well... adding those seems to make the button look weird

<img width="226" alt="Screen Shot 2021-02-10 at 3 48 47 PM" src="https://user-images.githubusercontent.com/19409/107570423-79b4a080-6bb7-11eb-8735-4ce22a650e32.png">